catalog/task/skopeo-copy/0.1/README.md

```diff
@@ -1,4 +1,4 @@
-# push-application-snapshot
```

Comment:
I know this isn't something you broke, but this should be `skopeo-copy` not `push-*` it was a copy paste error when it was originally added